src/actions/clusterActions.js

```diff
@@ -102,23 +102,19 @@ export function clusterInstallApp(app, clusterID) {
     });
 
     var appsApi = new GiantSwarmV4.AppsApi();
+    var appConfigsApi = new GiantSwarmV4.AppConfigsApi();
 
     var optionalCreateAppConfiguration = new Promise((resolve, reject) => {
       if (Object.keys(app.valuesYAML).length !== 0) {
         // If we have user config that we want to create, then
         // fire off the call to create it.
-        appsApi
+        appConfigsApi
           .createClusterAppConfig(scheme + ' ' + token, clusterID, app.name, {
             body: app.valuesYAML,
           })
           .then(() => {
-            // The call succeeded, resolve with a configmap
```

Comment:
the API does this linking of configmap to App CR automatically now, so that's why this step is gone.